### ISSUE TITLE
Unknown COMPOP "SOFT-LIGHT" and error with several lines with COMPFILTER with validate

### ISSUE THREAD

**Issue by LarsSchy** [geographika/mappyfile#150]
I found some errors in the handling of COMPOP and COMPFILTER.
The example I shared a couple of weeks ago with a working example with the composition pipeline: https://gist.github.com/LarsSchy/4aa75fd2a6bc7eb01a7cff9beb64b59a

I pasted this example into: https://mappyfile.geographika.net/#

Firstly I get an error when I push draw buttom:
loadLayerCompositer(): Expression parser error. Unknown COMPOP "SOFT-LIGHT"

The error goes away if I remove the COMPOP line.

However if I run the command line mappyfile validate on the same file it is ok.
I am running mappyfile --version --> 0.9.6

Secondly I also found out if I run mappyfile format it strips away all COMPFILTER rows except for the last one, if I have several COMPFILTERs stacked, that I would say is the normal case for creating a shadow with translate and blur combined.

**Comment by geographika** [comment]
Thanks for the report @LarsSchy - I wasn't aware multiple `COMPFILTER`s were allowed. I've pushed an update that hopefully fixes this and the the `COMPOP` issue, and I can now parse and generate your Mapfile at https://gist.githubusercontent.com/LarsSchy/4aa75fd2a6bc7eb01a7cff9beb64b59a/raw/a463a1b5e87e66bbbd1e14716138795484d3c649/layer_comp.map

Could I add this as one of the example maps at https://mappyfile.geographika.net/# ? I'd leave it as-is, with attribution in place. 

![image](https://user-images.githubusercontent.com/490840/161434006-9b6161fe-c357-4383-a330-446bd5a59eaf.png)

**Comment by LarsSchy** [comment]
Wow that was fast @geographika.  Yes it ok to use my example at https://mappyfile.geographika.net/#.  I would be honored if You did.

**Comment by LarsSchy** [comment]
I know that You showed me in the past that I could update mappyfile to master with:
pip install git+git://github.com/geographika/mappyfile@master -U
I seems like that that github is not allowing that nowadays.  What is the procedure now?

**Comment by geographika** [comment]
I think the https protocol should be used now, try the following to install from master:

```
pip install git+https://github.com/geographika/mappyfile.git -U
```

Great - thanks for letting me use the example, it will appear with the updated mappyfile in the next couple of weeks.

**Comment by LarsSchy** [comment]
Thanks, that works! and mappyfile format now works as expected.

**Comment by geographika** [comment]
@LarsSchy - the fix is contained in the new 0.9.7 release, and I added the composite example to https://mappyfile.geographika.net/#

![image](https://user-images.githubusercontent.com/490840/161448698-23016b55-6831-46a0-9c75-f3ca23a179d5.png)

Then press "Draw":

![image](https://user-images.githubusercontent.com/490840/161448713-b5be0fe4-83d9-4f50-a931-2ac85a66f35c.png)
